microsoft-edge/devtools-guide-chromium/whats-new/2022/09/devtools-105.md

```diff
@@ -0,0 +1,180 @@
+---
+title: What's new in DevTools (Microsoft Edge 105)
+description: "Focus Mode: Improved location controls for DevTools, Activity Bar, and Quick View. Improved reliability in Issues tool. And more."
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 09/01/2022
+---
+# What's New in DevTools (Microsoft Edge 105)
+
+[!INCLUDE [Microsoft Edge team note for top of What's New](../../includes/edge-whats-new-note.md)]
+
+<!-- ====================================================================== -->
+## Test your PWA's protocol handlers from the Application tool-->
+
+<!-- Rank: 1 -->
+<!-- Title: Test your PWA's protocol handlers from the Application tool  -->
+<!-- Subtitle: From the Manifest section of the Application tool, you can now provide custom protocols to launch your PWA. -->
+
+In Microsoft Edge 105, the Application tool now supports testing protocol handlers. Since Microsoft Edge 96, you've been able to define protocol handlers in the application manifest of your PWA. Now you can  test those protocols from the Application tool itself in DevTools if you have your PWA installed. 
```

Comment:
```suggestion
In Microsoft Edge 105, the Application tool now supports testing protocol handlers. Since Microsoft Edge 96, you've been able to define protocol handlers in the application manifest of your Progressive Web App (PWA). Now you can  test those protocols from the Application tool itself in DevTools if you have your PWA installed. 
```